Week3/homework/MY-step2-1-Hello_I_am_bar.js

```diff
@@ -1,15 +1,22 @@
 'use strict';
+/*sample*/
+// function doIt() {
+//   console.log('I am done');
+// }
+// setTimeout(doIt, 5000);
 
+//Test this code by copy-pasting the whole foo and bar functions to the browser console
 function foo(func) {
   // What to do here?
   // Replace this comment and the next line with your code
+  return ('Hi, I am AaaaT, who are you?');
   console.log(func);
```

Comment:
This exercise requires `foo` to take a function and invoke it. I'm not seeing that here. This should not be returning anything.

Reply:
` console.log(func);` can be removed

Reply:
function foo(func) {
  return ('Hi, I am AaaaT, who are you?');
  //console.log(func);
}

function bar() {
  console.log('Hello, I am bar!');
}
setTimeout(bar, 3000);
foo(bar);

Reply:
Maybe you did not see the function in pull request because it was already given by the excercise. I have tested it in the browser and it works. Turns out very cute me talking to the robot and the robot responding in 3 seconds :)

Reply:
Just had a look at the instructions again---

> 2.1 We saw that we can pass functions as arguments to other functions. Your task is to write a function that takes another function as an argument and runs it.

Currently, this code isn't doing that. Your foo should take a function and invoke it. You'll need to update your code to do precisely that. At the moment, you are just returning a string from foo, when it should be calling the function that is passed to it.